tests/test_permission_to_delete_is_supported.py

```
@@ -0,0 +1,112 @@
+
+    client.force_login(std_usr)
+    response = client.delete(f"/api/gated-authors/{author.id}")
+    assert response.status_code == status.HTTP_404_NOT_FOUND
+    assert models.Author.objects.filter(id=author.id).exists()
+
+
+@pytest.mark.django_db
+def test_delete_permitted_resource_with_unpermitted_related_resource_should_succeed(
+    client: Client,
+) -> None:
+    # TODO(phuongfi91): What about deleting with cascading against unpermitted related resources?
```

Ruff (0.8.2)

Missing issue link on the line following this TODO

(TD003)